github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRPullRequest.java

```diff
@@ -60,7 +62,7 @@ public GitHubPRPullRequest(GHPullRequest pr) throws IOException {
         userLogin = pr.getUser().getLogin();
         number = pr.getNumber();
         prUpdatedAt = pr.getUpdatedAt();
-        issueUpdatedAt = pr.getIssueUpdatedAt();
+        issueUpdatedAt = pr.getUpdatedAt();
```

Comment:
@jgangemi so just for your information. While other plugins even don't know difference. Issue has own update time and PR has own update time. It was used in light check before running any logic. Plus for comments you need issue time.